src/mapper.rs

```diff
@@ -3,9 +3,35 @@ use std::collections::HashSet;
 use std::fmt::{Error as FmtError, Write};
 use std::iter::FusedIterator;
 
+use crate::java;
 use crate::mapping::{ProguardMapping, ProguardRecord};
 use crate::stacktrace::{self, StackFrame, StackTrace, Throwable};
 
+/// A deobfuscated method signature.
+pub struct DeobfuscatedSignature {
+    parameters: Vec<String>,
+    return_type: String,
```

Comment:
IMO these fields can just be public. The type is just a collection of strings, it has no real invariants to uphold that need to be hidden. Then you don't need the accessor methods either.